core/server/api/canary/labels.js

```diff
@@ -0,0 +1,147 @@
+const Promise = require('bluebird');
+const common = require('../../lib/common');
+const models = require('../../models');
+
+const ALLOWED_INCLUDES = ['count.members'];
+
+module.exports = {
+    docName: 'labels',
+
+    browse: {
+        options: [
+            'include',
+            'filter',
+            'fields',
+            'limit',
+            'order',
+            'page',
+            'debug'
```

Comment:
@rishabhgrg why do we need the `debug` option here?

Reply:
@gargol No specific reason tbh, except that we use it in `tags` API as well so this just attempted to mimic it. 🤷‍♂

Reply:
Let's not include things that don't have any specific purpose. If we find a usecase to have this field in the future it's always non-breaking change to add instead of figuring out how to remove it :wink:

Reply:
Sure, thats sounds pretty reasonable, I'll patch it up 😄

Reply:
Done